extensions/jqt.menusheet.css

```diff
@@ -3,7 +3,6 @@
 #jqt > div.menusheet {
     -webkit-box-sizing: border-box; -moz-box-sizing: border-box; box-sizing: border-box;
     height: 100%;
-    width: 260px;
```

Comment:
@illucent, it should not be removed. The menu is not designed to be fully opened.

Please remove the changes to css.